src/makeInfinityAdder.js

```diff
@@ -4,9 +4,22 @@
  *
  * @return {function}
  */
+let sum = 0;
 
 function makeInfinityAdder() {
```

Comment:
give some parameter
you can write here like this - **makeInfinityAdder(x)**  with default value